hooks_template.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+import sys
+import os
+
+"""
+This file defines the required template for a given hooks-module to be used with the import_video.py 
+script. The `register_camera` and `post_video_content` functions are required, but the `set_hook_data`
+is optional and will only be called if it exists
+"""
+
+def set_hook_data(data_dict):
+    """
+    let the importer pass in arbitrary key-value pairs for use by the hooks program
+    (we tend to use this to accept plan data or user-account information)
```

Comment:
There is also reference to logging in the last line of this README so either document or remove that description of logging.